### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by joshlk**
@StellaAthena @sdtblck 

Added documentation for cluster stuff. Please feel free to edit as I'm dyslexic I'm sure its riddled with spelling mistakes and could be clearer 😅 .

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,10 +12,58 @@ TO DO
...
 
 TO
 
+## EleutherAI cluster
+
+### Using a cluster
+
+If you already have a EleutherAI cluster setup for you, test to see if the cluster is working:
+    
+1. Copy data from cold storage to the shared mount: `cp -R /mnt/ssd-0/megatron-3d/data /mnt/ssd-cluster/data`
```
Comment (sdtblck) [inline README.md:25]: I don't know if we should be putting commands that are specific to eleuther pods in the general readme - maybe a different branch for this?

Comment (joshlk) [reply README.md:25]: We already have EleutherAI cluster specific stuff in this repo... the kubernetes scripts and dockerimage ect. We could add this to a readme in the kubernetes folder?

**Comment by StellaAthena** [comment]
I have also been working on the documentation. Can you review the differences and merge the two branches?

https://github.com/EleutherAI/gpt-neox/tree/update-documentation

**Comment by joshlk** [comment]
Ive merged your branch and done some changes. Its ready to merge when your happy

**Review by StellaAthena — APPROVED** [review]

**Pull request merged**